frigate/const.py

```diff
@@ -1,14 +1,23 @@
-CONFIG_DIR = "/config"
-DEFAULT_DB_PATH = f"{CONFIG_DIR}/frigate.db"
-MODEL_CACHE_DIR = f"{CONFIG_DIR}/model_cache"
-BASE_DIR = "/media/frigate"
-CLIPS_DIR = f"{BASE_DIR}/clips"
-RECORD_DIR = f"{BASE_DIR}/recordings"
-EXPORT_DIR = f"{BASE_DIR}/exports"
-BIRDSEYE_PIPE = "/tmp/cache/birdseye"
-CACHE_DIR = "/tmp/cache"
+from pathlib import Path
+
+INSTALL_DIR = Path("/opt/frigate")
+
+CONFIG_DIR = Path("/config")
+DEFAULT_DB_PATH = CONFIG_DIR / "frigate.db"
+MODEL_CACHE_DIR = CONFIG_DIR / "model_cache"
```

Comment:
I feel like this should be below CACHE_DIR, wdyt?

Reply:
if you are trying to go for alphabetical then cache should probably go at the top, but I think it is fine as is tbh

Reply:
No, I meant putting the model cache below `/tmp/cache`. But I can also sort the vars.

Reply:
I don't think that makes sense because it has nothing to do with /cache

Reply:
let's just leave as is